auth-foundation/src/main/java/com/okta/authfoundation/client/EncryptionTokenProvider.kt

```diff
@@ -21,49 +21,33 @@ import androidx.datastore.preferences.core.Preferences
 import androidx.datastore.preferences.core.edit
 import androidx.datastore.preferences.core.stringPreferencesKey
 import androidx.datastore.preferences.preferencesDataStore
-import com.okta.authfoundation.InternalAuthFoundationApi
 import com.okta.authfoundation.util.AesEncryptionHandler
 import kotlinx.coroutines.flow.firstOrNull
 import java.util.UUID
 
-@InternalAuthFoundationApi
-class DeviceTokenProvider(
+internal class EncryptionTokenProvider(
```

Comment:
I decided to use a completely different token for DB encryption compared to the one used for DT cookie. It seemed like an unnecessary risk.